src/AudioUnit/AudioUnit.cs

```diff
@@ -1032,6 +1085,10 @@ class AURenderCallbackStrct {
 		static extern AudioUnitStatus AudioUnitSetParameter (IntPtr inUnit, AudioUnitParameterType inID, AudioUnitScopeType inScope,
 			uint inElement, float inValue, uint inBufferOffsetInFrames);
 
+		[iOS (2,0)]
```

Comment:
^ remove, undecorated API are assumed to be part of our minimum support (so start with 6.x).